src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.PWMSparkMax;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+/**
+ * this subsystem takes the balls from the field to the funnel.
+ */
+public class Intake extends SubsystemBase {
+    private PWMSparkMax motor = new PWMSparkMax(Ports.Intake.MOTOR);
+    private Solenoid solenoidR = new Solenoid(Ports.Intake.SOLENOID_RIGHT);
+    private Solenoid solenoidL = new Solenoid(Ports.Intake.SOLENOID_LEFT);

```

Comment:
The naming does not match the convention. Also, the word "solenoid" is an uninteresting implementation detail. I think "retractor" will be better:

```suggestion
    private Solenoid rightRetractor = new Solenoid(Ports.Intake.SOLENOID_RIGHT);
    private Solenoid leftRetractor = new Solenoid(Ports.Intake.SOLENOID_LEFT);
```
However, there is only one solenoid for both pistons, so you can be even more concise:

```suggestion
    private Solenoid retractor = new Solenoid(Ports.Intake.RETRACTOR_SOLENOID);

```